client/src/components/FormCheckboxField/FormCheckboxField.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { FieldProps } from 'formik';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import {FormControlLabelProps} from '@material-ui/core/FormControlLabel/FormControlLabel';
+
+
+const FormCheckboxField: React.FC<FieldProps & FormControlLabelProps> = ({ field, label }) => {
```

Comment:
If this is checkbox, we all know that it is relating to form. Maybe just `checkbox` or something simple like that will be good?